queries.sh

```diff
@@ -0,0 +1,117 @@
+# Testing parent/child, remove before merging
```

Comment:
This will be removed in final version?

Reply:
Yep, I'll remove both `queries.sh` and `manual.rb`, I'll keep them somewhere locally